### ISSUE TITLE
Keyboard shortcut regression

### ISSUE THREAD

**Issue by kelderek** [notepad-plus-plus/notepad-plus-plus#2595]
### Description of the Issue
In 7.2.1 32-bit pressing ctrl-H to bring up replace dialog then alt-A to replace all performs the find all in current document action and not the replace all action.  Clicking replace all works as expected.

### Steps to Reproduce the Issue
1.  Press Ctrl-H to bring up replace dialog
2.  Enter text to search for and replacement text
3.  Press Alt-A to do a Replace All

### Expected Behavior
Replace all search text with replacement in current document

### Actual Behavior
Finds all search text in current document and brings up Find result lower panel.  No replacements are made

### Debug Information
Notepad++ v7.2.1   (32-bit)
Build time : Nov 20 2016 - 22:42:51
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10
Plugins : ComparePlugin.dll mimeTools.dll NppConverter.dll NppExport.dll PluginManager.dll XMLTools.dll

**Comment by SinghRajenM** [comment]
I feel this regression is because of #2391 

``` C++
     PUSHBUTTON      "Find &All in Current Document",IDC_FINDALL_CURRENTFILE,268,80,90,21,BS_MULTILINE
     PUSHBUTTON      "&Replace",IDREPLACE,268,38,90,14
     PUSHBUTTON      "Replace &All",IDREPLACEALL,268,56,90,14
```

Accelerator key 'A' conflicts for "Find &All in Current Document" and "Replace &All". "Find &All in Current Document" comes first in tab order which get advantage of it.

**Comment by dilijev** [comment]
Oops, that seems to be the case. I'm surprised that buttons on different tabs interfere with each other?

**Comment by dilijev** [comment]
Curious, why is there no problem with the overlap of the assigned key for
`Find in All &Opened Documents` and
`Replace All in All &Opened Documents`?

**Comment by dail8859** [comment]
Closed by 1ce7df0408abf56101158dd4eaba008b1acf2da1
